tests/rustfmt/main.rs

```diff
@@ -106,3 +106,13 @@ fn inline_config() {
             && contains("format_strings = true")
     );
 }
+
+#[test]
+fn rustfmt_help() {
+    let args = [
+        "--help",
+    ];
+    let (stdout, _) = rustfmt(&args);
+    assert!(stdout.contains(&format!("Format Rust code")));
+    assert!(stdout.contains(&format!("usage: rustfmt [options] <file>...")));
```

Comment:
To show that we're not including the path of the rustfmt executable would it not be easier to do a single check for the string ``"Format Rust code\n\nusage: rustfmt [options] <file>..."``?